test/05LessonFiveTest.js

```diff
@@ -4,11 +4,24 @@ var assert = require('assert');
 
 function loopIt() {
```

Comment:
Remember that your function names should give a clear idea of what the function is for. 'loopIt', while conveying to me that a loop is incorporated in some way, does not convey much more. As this appears to create an array of 100 integers, a better name would be 'generateArrayOfInts()'